src/index.html

```diff
@@ -7,8 +7,41 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1>Moyo header</h1>
+<header class="header">
+  <a href="#" class="logo--link">
+    <img class="logo--image" src="images/logo.png" alt="MOYO-logo">
+  </a>
+  <nav class="nav">
+    <ul class="nav--list">
+      <li class="nav--item">
+        <a class="nav--link is-active" href="#">Apple</a>
+      </li>
+      <li class="nav--item">
+        <a class="nav--link" href="#">Samsung</a>
+      </li>
+      <li class="nav--item">
+        <a class="nav--link" href="#">Смартфоны и телефоны</a>
+      </li>
+      <li class="nav--item">
+        <a class="nav--link" data-qa="hover" href="#">Ноутбуки и компьютеры</a>
+      </li>
+      <li class="nav--item">
+        <a class="nav--link" href="#">Гаджеты</a>
+      </li>
+      <li class="nav--item">
+        <a class="nav--link" href="#">Планшеты</a>
+      </li>
+      <li class="nav--item">
+        <a class="nav--link" href="#">Фото</a>
+      </li>
+      <li class="nav--item">
```

Comment:
it must be element, not modifier, write elements via __item

Reply:
fixed. solution uploaded